core/controlplane/config/templates/cluster.yaml

```diff
@@ -491,14 +485,8 @@ worker:
 #      ephemeralImageStorage:
 #        enabled: true
 #
-#      # When enabled this will install kiam using the repo from `kiamImage` (ref https://github.com/uswitch/kiam)
-#      # It will also grant sts:assumeRole permission to the IAM role for worker nodes in this pool.
-#      # This will use SSL certificates generated during `kube-aws render credentials`, ensure this has been run with kube-aws v0.9.9+
-#      kiamSupport:
-#        enabled: true
```

Comment:
Ah, yes. Currently we have a nodeSelector to stick kiam server to controller nodes. So no need for kiamSupport in worker node pools. Good catch 👍